### PR TITLE
Parseq/rr

### PR DESCRIPTION
**Pull request by shubhanshu-mani**

### CODE REVIEW

File: src/Presto/Backend/Language/Types/EitherEx.purs [hunk 1]
```diff
@@ -36,6 +36,7 @@ import Prelude
 import Data.Either (Either(..), either)
 import Data.Foreign.Class (class Decode, class Encode)
 import Data.Generic.Rep (class Generic)
+--import Presto.Backend.Language.Types.ParSequence (ParError(..))
```
Comment (jarnura) [inline src/Presto/Backend/Language/Types/EitherEx.purs:39]: can you remove the commented code

File: src/Presto/Backend/Language/Types/EitherEx.purs [hunk 2]
```diff
@@ -56,6 +57,9 @@ fromEitherEx = eitherEx Left Right
 toEitherEx :: forall l r. Either l r -> EitherEx l r
 toEitherEx = either LeftEx RightEx
 
+-- toParEitherEx :: forall l r. Either l r -> Either ParError r
```
Comment (jarnura) [inline src/Presto/Backend/Language/Types/EitherEx.purs:60]: here also

File: src/Presto/Backend/Playback/Entries.purs [hunk 3]
```diff
@@ -156,17 +158,17 @@ mkCallAPIEntry
   -> CallAPIEntry
 mkCallAPIEntry jReqF aRes = CallAPIEntry
   { jsonRequest : jReqF unit
-  --, jsonResult  : encode <$> aRes
+  , jsonResult  : encode <$> aRes
```
Comment (jarnura) [inline src/Presto/Backend/Playback/Entries.purs:161]: why this change bro

**Review by jarnura — CHANGES_REQUESTED** [review]

File: src/Presto/Backend/Playback/Entries.purs (hunk 3 above)
Comment (shubhanshu-mani) [reply src/Presto/Backend/Playback/Entries.purs:161]: This change was done by mistake

**Pull request closed without merging**